Src/FluentAssertions/Formatting/DateTimeOffsetValueFormatter.cs

```diff
@@ -70,12 +72,15 @@ public void Format(object value, FormattedObjectGraph formattedGraph, Formatting
                 formattedGraph.AddFragment(" +");
                 formatChild("offset", dateTimeOffset.Offset, formattedGraph);
             }
-
-            if (dateTimeOffset.Offset < TimeSpan.Zero)
+            else if (dateTimeOffset.Offset < TimeSpan.Zero)
             {
                 formattedGraph.AddFragment(" ");
                 formatChild("offset", dateTimeOffset.Offset, formattedGraph);
             }
+            else if (significantOffset && (hasDate || hasTime))
+            {
+                formattedGraph.AddFragment(" UTC");
+            }
```

Comment:
🤔 What happens if the last `else if` is not happening?

Reply:
That would leave us in the original path. No offset appended/prepended.

Reply:
I would add an `else` with a comment explaining that. Avoids unnecessary frowns later on.